packages/utilities/src/utilities/focuswrap/focuswrap.js

```diff
@@ -15,7 +15,7 @@ import on from 'carbon-components/es/globals/js/misc/on';
  *   The focus sentinel nodes.
  *   If these nodes gets focus, we see it as focus went out of the `element`.
  * @param {string} [eventRequestFocusWrap=dds-request-focus-wrap] The event name.
- * @returns {Handle} The handle to remove the event handler.
+ * @returns {Function} The handle to remove the event handler.
```

Comment:
Actually this function returns an object instead of a function. What kind of ESLint error did you get?